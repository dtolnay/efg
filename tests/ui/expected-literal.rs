use efg::efg;

#[efg(feature = STD)]
struct S;

fn main() {}
