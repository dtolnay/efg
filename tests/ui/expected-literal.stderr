error: unexpected token, expected a string literal
 --> tests/ui/expected-literal.rs:3:17
  |
3 | #[efg(feature = STD)]
  |                 ^^^
